Widget/Fitness/FitnessWidget.swift

```diff
@@ -0,0 +1,238 @@
+    var body: some View {
+        VStack {
+            Text(rooms[0].name)
+                .font(.system(size: 13, weight: .semibold))
+                .padding(.top)
+            Text(getOpenString(room: rooms[0], showbusystring: false))
+                .font(.system(size: 11))
+                .foregroundStyle(isOpen(room: rooms[0]) ? .green : .blue)
+                .multilineTextAlignment(.center)
+                .fixedSize(horizontal: false, vertical: true)
+            MeterView(current: isOpen(room: rooms[0]) ? rooms[0].capacity : 0, maximum: 100.0, style: Color.blue, lineWidth: 6) {
+                VStack {
+                    Text("\(isOpen(room: rooms[0]) ? rooms[0].capacity : 0, specifier: "%.2f")%")
+                        .fontWeight(.bold)
+                    Text("capacity")
+                        .font(.system(size: 10, weight: .light, design: .default))
+                    
+                }
+            }
+            .frame(width: 90, height: 90)
+            
+            Text(lastUpdatedFormattedTime(date: rooms[0].last_updated))
+                .font(.system(size: 9))
+                .foregroundStyle(.gray)
+            
+            Spacer()
+        }.widgetPadding()
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+    
+}
+
+private struct FitnessWidgetMediumView: View {
+    var rooms: [FitnessRoom]
+    var configuration: ConfigureFitnessWidgetIntent.Configuration
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Text(rooms[0].name)
+                    .font(.system(size: 12, weight: .semibold))
+                    .padding(.top)
+                
+                Text("")
+                
+                Text(getOpenString(room: rooms[0], showbusystring: true))
+                    .font(.system(size: 12))
+                    .foregroundStyle(isOpen(room: rooms[0]) ? .green : .blue)
+                    .multilineTextAlignment(.center)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .padding(.top)
+            }
+            .padding([.trailing, .leading], 10)
+            HStack {
+                VStack {
+                    MeterView(current: isOpen(room: rooms[0]) ? rooms[0].capacity : 0, maximum: 100.0, style: Color.blue, lineWidth: 6) {
+                        VStack {
+                            Text("\(isOpen(room: rooms[0]) ? rooms[0].capacity : 0, specifier: "%.2f")%")
+                                .fontWeight(.bold)
+                            Text("capacity")
+                                .font(.system(size: 10, weight: .light, design: .default))
+                            
+                        }
+                    }
+                    .frame(width: 90, height: 90)
+                    
+                    Text(lastUpdatedFormattedTime(date: rooms[0].last_updated))
+                        .font(.system(size: 9))
+                        .foregroundStyle(.gray)
```

Comment:
Going to nitpick a bit here
```suggestion
                        .foregroundStyle(.secondary)
```